pkg/relay/ethereum/tx.go

```diff
@@ -262,3 +280,14 @@ func (c *Chain) TxAcknowledgement(opts *bind.TransactOpts, msg *chantypes.MsgAck
 		ProofHeight:     pbToHandlerHeight(msg.ProofHeight),
 	})
 }
+
+func (c *Chain) GetChannelLogger() *log.RelayLogger {
+	logger := GetModuleLogger()
+	if c.Path() == nil {
+		return logger
+	}
+	chainID := c.Path().ChainID
+	portID := c.Path().PortID
+	channelID := c.Path().ChannelID
+	return logger.WithChannel(chainID, portID, channelID)
+}
```

Comment:
The logger created by `GetChannelLogger` is only suitable in the context of an established channel.
The following are examples of **NOT** suitable use cases.
- SendMsgs
    - This function is used for CreateClient or ConnectionOpenXxx or ChannelOpenXxx ...
- TxOpts
    - This function is used for sending all types of transactions, including ones inappropriate for `GetChannelLogger`

Reply:
Only use channellogger for events

https://github.com/datachainlab/ethereum-ibc-relay-chain/pull/13/commits/260d6c02f22ed6a95b057e7df026522d351b59d6

Reply:
@dongrie thank you. please also use `GetChannelLogger` for `QueryUnreceivedXxx` and `QueryUnfinalizedXxx`.

Reply:
https://github.com/datachainlab/ethereum-ibc-relay-chain/pull/13/commits/ec25e08df97694c8e6dc56ccb9fe9bbc751aad59
Fixed